design/admin/templates/ajaxuploader/location.tpl

```diff
@@ -3,6 +3,20 @@
     <fieldset>
         <legend>{"Step 2/3: Choose a location for the new '%class' object"|i18n( 'design/admin/ajaxuploader', '', hash( '%class', $class.name|wash() ) )}</legend>
         <p>{'<em>%file</em> has successfully been uploaded.'|i18n( 'design/admin/ajaxupload', '', hash( '%file', $file.original_filename|wash() ) )}</p>
+        {if $class.identifier|eq( 'image' )}
+            {def $alt_text_required = ''}
+            {foreach $class.data_map as $attribute}
+                {if $attribute.data_type_string|eq( 'ezimage' )}
+                    {if $attribute.data_int2}
+                        {set $alt_text_required = ' (required)'}
+                    {/if}
+                {/if}
+            {/foreach}
+            <p>
+                <label for="alt_text_entry">Alternative text{$alt_text_required}:</label>
+                <input{if $alt_text_required} required class="input-required"{/if} id="UploadAlternativeText" name="UploadAlternativeText" size="70" maxlength="150">
```

Comment:
Regarding the maxlength:
- I'm thinking that there needs to be a setting here.  I think I just hard-coded that.  I thought it dangerous to let it be open ended.  I am only limiting it here and nowhere else.  I didn't address anything for the content of the alt tag itself as we decided that should be a separate modification and I thought the length of the alt attribute would fall under that.  Otherwise, I think we would have to control the allowable size at the same level as the mandatory alt tag so it is controlled at the class attribute level.
Regarding the size
- Pretty sure I just copied the size of the other input box that was in use.